This patch works as a supplement for cronet_appreduce_workaround.patch.
See crbug.com/1087646 on why this is needed.
For more information, also see cronet_appreduce_workaround.patch.

--- base/android/proguard/chromium_code.flags	2020-05-29 11:34:48.748109273 -0700
+++ modified.flags	2020-05-29 12:20:30.692111915 -0700
@@ -34,13 +34,13 @@
 -keep @org.chromium.base.annotations.UsedByReflection class ** {}
 
 # Keeps for method level annotations.
--keepclasseswithmembers,allowaccessmodification class ** {
+-keepclasseswithmembers class ** {
   @org.chromium.base.annotations.AccessedByNative <fields>;
 }
--keepclasseswithmembers,includedescriptorclasses,allowaccessmodification class ** {
+-keepclasseswithmembers,includedescriptorclasses class ** {
   @org.chromium.base.annotations.CalledByNative <methods>;
 }
--keepclasseswithmembers,includedescriptorclasses,allowaccessmodification class ** {
+-keepclasseswithmembers,includedescriptorclasses class ** {
   @org.chromium.base.annotations.CalledByNativeUnchecked <methods>;
 }
 -keepclasseswithmembers class ** {
